app/coffeescripts/handlebars_helpers.coffee

```diff
@@ -150,6 +150,16 @@ define [
       real_min_str = (if real_minutes < 10 then "0" + real_minutes else real_minutes)
       "#{hours}:#{real_min_str}"
 
+    # Convert the total amount of minutes into a readable duration format.
+    durationToString : (minutes) ->
+      hours = Math.floor(minutes / 60)
+      real_minutes = minutes % 60
```

Comment:
How come you are passing in minutes but then you have to convert to real minutes? Do you mean that the param is actually seconds? I would reword this variable because this doesn't make sense to me. I would also add some comments on what is getting passed in and document the math you are doing

Reply:
Yeah, agree. It is kind of confusing in the way it is right now. And although it is based on the function written a few lines above (https://github.com/instructure/canvas-lms/blob/stable/app/coffeescripts/handlebars_helpers.coffee#L147) rephrasing and adding comments may help to make it clearer.